pip/main.ts

```diff
@@ -73,13 +73,15 @@ async function checkDryRun() {
       net: true,
     },
     stdout: 'piped',
+    stderr: 'piped',
   });
 
   // Ensure dry-run was successful.
   if (!status.success) {
+    console.log(status);
```

Comment:
Is this log here intentional? Seems like leftover debug information to me.

Reply:
debug stuff, removed!